content/en/docs/hertz/tutorials/toolkit/usage/usage-thrift.md

```diff
@@ -42,6 +42,20 @@ go mod edit -replace github.com/apache/thrift=github.com/apache/thrift@v0.13.0
 go mod tidy
 ```
 
+```bash
+// Execute is not under GOPATH
+//option 1,you do not have a go.mod, add go mod name after - module
```

Comment:
‘- module’ -> `-module`?

Try to use code block.

Reply:
Get, then just remove the extra blank between - and module

Reply:
I have removed it. You can see Files changed